feat: Make live-reloading explicit for codejail

timmc-edx opened the pull request:
This goes with https://github.com/edx/public-dockerfiles/pull/92 which switches the default CMD from runserver to gunicorn; we want to keep runserver.

----

I've completed each of the following or determined they are not applicable:

- [x] Made a plan to communicate any major developer interface changes (or N/A)

jbarciauskas commented:
Oh great. Waiting for CI to run but glad to hear this might address the CPU usage John reported

clintonb commented:
The build is failing because uninstall edxval causes Django to stop. We should stop uninstalling packages in this manner.

clintonb commented:
Related: https://github.com/edx/edx-platform/pull/14947

timmc-edx committed:
feat: Make live-reloading explicit for codejail

This goes with https://github.com/edx/public-dockerfiles/pull/92 which
switches the default CMD from runserver to gunicorn; we want to keep
runserver.
